### DISCUSSION TITLE
TriliumNext Notes (original Trillium Notes is in maintenance mode)

### DISCUSSION THREAD

**Discussion by fmagliaro** [community-scripts/ProxmoxVE#1068] [Request script] [answered]
### Application Name

TriliumNext Notes

### Website

https://github.com/TriliumNext/Notes

### Description

TriliumNext Notes is an open-source, cross-platform hierarchical note taking application with focus on building large personal knowledge bases.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by fmagliaro** [comment]
Hello,  The original Trillium script you have is in maintenance mode.  
https://github.com/zadam/trilium#trilium-notes
"Trilium is in maintenance mode - see details in https://github.com/zadam/trilium/issues/4620"

**Comment by knario** [comment, marked as answer]
The script in the community already contains trilium next, try to install it.
